homeassistant/components/wemo/light.py

```diff
@@ -141,16 +138,18 @@ def color_temp(self) -> int | None:
         """Return the color temperature of this light in mireds."""
         return cast(Optional[int], self.light.state.get("temperature_mireds"))
 
+    @property
+    def color_mode(self) -> ColorMode:
+        """Return the color mode of the light."""
+        if self.hs_color:
```

Comment:
If we assume the existing code is correct, we should do like this to avoid an unnecessary call to `color_util.color_xy_to_hs`:
```suggestion
        if self.light.state.get("color_xy"):
```
```suggestion
        if self.hs_color:
```